### PR TITLE
Updates to README and CHANGELOG

### PR DESCRIPTION
**Pull request by tcld**
I only updated what I knew of; might have overlooked some things.
(I also reordered the bullet points in the changelog a bit - changes first, then additions, then removals.)

### CODE REVIEW

**Comment by ftomassetti** [comment]
The build stalled, restarting it

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
 Version 0.19-UNRELEASED
 
-* Added export support using libgdal
-* Force rivers and grayscale_heightmap to PNG to avoid ambiguity of providing a 
-filename which can introduce input error.
+* Force rivers and grayscale_heightmap to PNG to avoid ambiguity of providing a filename which can introduce input error.
```
Comment (ftomassetti) [inline CHANGELOG.md]: Can we rephrase this one?

**Comment by ftomassetti** [comment]
It looks good to me, just a minor comment added. And finally you had yourself to the contributors after just 68 commits :D

File: CHANGELOG.md (hunk 1 above)
Comment (tcld) [inline CHANGELOG.md]: Probably. But I didn't come up with this sentence.^^

How about..."force blabla to PNG" only? I am not sure if somebody who reads the changelog needs to know about all the details.
Other suggestions welcome. :P

Comment (ftomassetti) [inline CHANGELOG.md]: "Rivers and grayscale heightmaps are generated as png images" ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
 Version 0.19-UNRELEASED
 
-* Added export support using libgdal
-* Force rivers and grayscale_heightmap to PNG to avoid ambiguity of providing a 
-filename which can introduce input error.
+* Rivers and grayscale_heightmaps are generated as PNG images, filenames no longer have to be provided.
```
Comment (tcld) [inline CHANGELOG.md]: Is this better?

File: README.md [hunk 3]
```diff
@@ -341,6 +341,8 @@ We would like to thank you great people who helped us while working on WorldEngi
 
 * [Joshua Coppola](https://github.com/pangal) implemented the satellite view. Thanks a lot, it looks gorgeous!
 
+* [Stephan](https://github.com/tcld) made WorldEngine make heavy use of numpy, helping to speed up the generation. He also made world-generation much more reproducible and helped improving compatibility wih Python 3.
```
Comment (tcld) [inline README.md]: Oh, there is a typo in the final "with".

File: CHANGELOG.md (hunk 2 above)
Comment (ftomassetti) [inline CHANGELOG.md]: Yes!

**Comment by psi29a** [comment]
Looks good!

**Pull request merged**